### ISSUE TITLE
ActiveSupport >6.0 does not support Ruby 2.4

### ISSUE THREAD

**Issue by Schwad** [inspec/train#598]
This is a hard break, ActiveSupport will not work with Ruby 2.4. As long as we support Ruby 2.4, we will need to pin ActiveSupport < 6.0

**Comment by clintoncwolfe** [comment]
https://github.com/inspec/train/blob/d71ed3775933bb9c14a0ef366c53ddd32121b863/train.gemspec#L34

**Comment by Vasu1105** [comment]
activesupport > 6.0 for ruby 2.4 error
```
bundle install
Fetching gem metadata from https://rubygems.org/.......
Fetching gem metadata from https://rubygems.org/.
Resolving dependencies...
Bundler found conflicting requirements for the Ruby version:
  In Gemfile:
    Ruby

    train was resolved to 3.7.0, which depends on
      activesupport (>= 5.2.4.3, > 6.0.0) was resolved to 6.1.3.1, which depends on
        Ruby (>= 2.5.0)

    license_finder was resolved to 6.0.0, which depends on
      bundler was resolved to 2.1.4, which depends on
        Ruby (>= 2.3.0)

    train was resolved to 3.7.0, which depends on
      Ruby (>= 2.4)
```
